lib/credential/domain/use_cases/save_claims_use_case.dart

```diff
@@ -8,14 +8,14 @@ import '../../../iden3comm/domain/use_cases/get_fetch_requests_use_case.dart';
 
 class SaveClaimsParam {
   final List<ClaimEntity> claims;
-  final String did;
-  final int profileNonce;
+  final String genesisDid;
+  //final BigInt profileNonce;
```

Comment:
Commented code, we don't use it anymore (can remove)?

Reply:
we will use it for filtering by profile nonce, will need to test it and see if I can add it to this release